docker/app/Dockerfile

```
@@ -45,39 +55,80 @@ ENV RAILS_LOG_TO_STDOUT=true
 ENV RAILS_SERVE_STATIC_FILES=true
 ENV LANG=en_US.UTF-8
 ENV LANGUAGE=en_US.UTF-8
-# Tell Puppeteer to skip installing Chrome. We'll be using the installed package.
-ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
-ENV PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium-browser
-ENV GROVER_NO_SANDBOX=true
-ENV CHROMIUM_PATH=/usr/bin/chromium-browser
 
 # Allow tls connection to database with verification
 COPY docker/app/us-east-1-bundle.pem  /etc/ssl/certs/us-east-1-bundle.pem
-# Deprecated and soon to expire
-COPY docker/app/rds-combined-ca-bundle.pem /etc/ssl/certs/rds-combined-ca-bundle.pem
 
-SHELL [ "/bin/ash", "-o", "pipefail", "-c" ]
-RUN addgroup -g ${GROUP_ID} app-user || echo ok \
-  && grep ":${GROUP_ID}:" /etc/group | cut -f1 -d: > /.group_name \
-  && adduser -u ${USER_ID} -G "$(cat /.group_name)" -h /home/app-user -D app-user \
+RUN groupadd -g ${GROUP_ID} app-user \
+  && useradd -l -u ${USER_ID} -g app-user -s /bin/bash -d /home/app-user -m app-user \
   && mkdir /app /bundle \
-  && chown app-user:${GROUP_ID} /bundle /app
-
-RUN apk update \
-  && apk upgrade --no-cache \
-  && apk add --no-cache \
-  nodejs yarn \
-  aws-cli \
-  tzdata \
-  git \
+  && chown app-user:app-user /bundle /app
+
+# Add PostgreSQL official repository, NodeJS and Yarn repositories
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends curl ca-certificates lsb-release gnupg \
+  && sh -c 'echo "deb https://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+  && curl -fsSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor -o /etc/apt/trusted.gpg.d/postgresql.gpg \
+  && curl -fsSL https://deb.nodesource.com/setup_18.x | bash -s -- -y \
+  && apt-get update \
+  && apt-get remove -y nodejs \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get upgrade -y \
+  && apt-get install -y --no-install-recommends \
+  awscli \
   bash \
-  linux-headers \
-  icu icu-dev \
-  curl libcurl curl-dev \
-  libmagic file-dev file libffi-dev yaml-dev \
-  build-base libxml2-dev libxslt-dev postgresql-dev tmux postgresql shared-mime-info \
-  tar xz \
-  && git config --global --add safe.directory /app
+  build-essential \
+  dbus \
+  dbus-user-session \
+  file \
+  fonts-dejavu \
+  fonts-droid-fallback \
+  fonts-freefont-ttf \
+  fonts-liberation \
+  git \
+  gnupg-agent \
+  gpg \
+  imagemagick \
+  libcurl4 \
+  libcurl4-openssl-dev \
+  libffi-dev \
+  libfreetype-dev \
+  libfreetype6 \
+  libgeos-dev \
+  libglib2.0-0 \
+  libharfbuzz-dev \
+  libheif1 \
+  libicu-dev \
+  libmagic-dev \
+  libmagic1 \
+  libpq-dev \
+  libx11-6 \
+  libxext6 \
+  libxml2-dev \
+  libxrender1 \
+  libxslt1-dev \
+  libyaml-dev \
+  locales-all \
```

If we're trying to be more frugal, looks like , looks like locales-all is north of 200MB. We could do:
```suggestion
  locales \
```
but then we'd probably also need to set the locale too, possibly with this (not tested)
```
RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen \
  && locale-gen \
  && update-locale LANG=en_US.UTF-8
```

Chromium is also big but looks like it's been removed. Also awscli is 95MB for some reason but we probably need that for something. Looking at

```bash
dpkg-query -Wf='${Installed-Size}\t${Package}\n' | sort -n
```